docs/reference/cluster/stats.asciidoc

```diff
@@ -1307,6 +1307,125 @@ Number of selected nodes using the distribution flavor and file type.
 =====
 ====
 
+`snapshots`::
+(object)
+Contains statistics about the <<snapshot-restore,snapshot>> activity in the cluster.
++
+.Properties of `snapshots`
+[%collapsible%open]
+=====
+
+`current_counts`:::
+(object)
+Contains statistics which report the numbers of various ongoing snapshot activities in the cluster.
++
+.Properties of `current_counts`
```

Comment:
I want to say that they're `counts` too here, rather than calling the fields `snapshots_count`, `shard_snapshots_count`, and so on. Moreover we use `current` in this way in other stats APIs.